src/api/controllers/articleController.js

```diff
@@ -300,4 +332,33 @@ export default class ArticleController {
       message: `${title} has been shared successfully on ${sharedOn}, Thanks`
     });
   }
+
+  /**
+   * This function gets all articles that belongs to a specific category
+   *
+   * @static
+   * @param {Object} req the request
+   * @param {Object} res the response to be sent
+   * @memberof Articles
+   * @returns {Object} res
+   */
+  static async getArticleByCategory(req, res) {
```

Comment:
@murediane since this is fetching many articles, it should be ` getArticlesByCategory`

Reply:
working on it